server/src/api/roster/roster.controller.ts

```diff
@@ -326,9 +326,17 @@ class RosterController {
     res.json(entry);
   }
 
+  async savePhoneNumber(edipi: string, phoneNumber: string) {
+    const roster = await Roster.findOne({ where: { edipi }});
```

Comment:
The same `edipi` can potentially be in the roster table multiple times, since an individual can be on multiple units. So I think this should be a `Roster.find` call that updates potentially multiple roster entries.

Reply:
Hmm....I did not know this!

Reply:
done